distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -413,4 +421,12 @@ public void testConfigurationFileWithInvalidFormat() throws IOException {
         invalidConfig.addConfigurationFile();
         invalidConfig.assertFileMovedToFailedDirectory();
     }
+
```

Comment:
This test already checks the exporting of files to the export directory, but we may need the opposite test (exporting nothing).  Not sure about the other writing the other test though, I'll try using ```File.setReadOnly``` but I'm reading some articles that say this might not work on all OS.